Changes

```diff
@@ -626,6 +626,12 @@ Working version
 - #12046: Flush stderr when tracing the parser
   (Hugo Heuzard, review by David Allsopp and Nicolás Ojeda Bär)
 
+- #12075: auto-detect whether `ar` support @FILE arguments at
+  configure-time to avoid using this feature with toolchains/systems
+  that do not support it (eg BSD/Darwin).
```

Comment:
```suggestion
  that do not support it (e.g. FreeBSD/Darwin).
```

nit: BSD isn't a single OS these days; it's FreeBSD you're referring to I think.

Reply:
Thanks, fixed.